lightning/src/util/test_utils.rs

```diff
@@ -501,6 +501,10 @@ impl<Signer: sign::ecdsa::WriteableEcdsaChannelSigner> chainmonitor::Persist<Sig
 		}
 		res
 	}
+
+	fn prune_persisted_channel(&self, _funding_txo: OutPoint) -> bool {
+		false
```

Comment:
If I do `self.persister.prune..` here I get some type errors.. didnt have time today to look into it any further